src/App.tsx

```diff
@@ -1,204 +1,106 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useRef, useEffect, useState, useCallback } from 'react';
 import { UserWarning } from './UserWarning';
 import { USER_ID } from './api/todos';
+import * as todosServices from '../src/api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { Error } from './components/Error/Error';
+import { FilterType } from './types/FilterType';
+
+
+
+enum ErrorMessages {
+  loadTodos = 'Unable to load todos',
+  emptyTitle = 'Title should not be empty',
+  addTodo = 'Unable to add a todo',
+  deleteTodo = 'Unable to delete a todo',
+  updateTodo = 'Unable to update a todo',
+}
+
 
 export const App: React.FC = () => {
   if (!USER_ID) {
     return <UserWarning />;
```

Comment:
<img width="652" alt="image" src="https://github.com/user-attachments/assets/cf3aebb5-3a3b-4c95-b1ce-f450d7b63af7" />
This is an important comment; we must use hooks before all checks